### PR TITLE
"Emotion" page fix; Metatags deleted

### PR DESCRIPTION
**Pull request by nichitatrifan**

### CODE REVIEW

File: src/sass/section.scss [hunk 1]
```diff
@@ -102,7 +102,6 @@
 .ms-back {
```
Comment (protyagov) [inline src/sass/section.scss:102]: Delete this class completely.

File: src/landing/p7-emotion.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default function P7Emotion() {
         <NavigationBar />
 
         <div className="ms-base-page pb-5 ms-back">
```
Comment (protyagov) [inline src/landing/p7-emotion.tsx:11]: Class ms-back can be deleted, it is not used.

**Review by protyagov — CHANGES_REQUESTED** [review]
Mostly good, we just need to remove ms-back class

**Comment by nichitatrifan** [comment]
deleted the 'ms-black' class

**Pull request merged**